tests/CMakeLists.txt

```diff
@@ -28,7 +28,8 @@
 #
 MACRO(ADD_TESTCASE NAME)
   ADD_UNIT_TEST(${NAME} ${NAME}.cc)
-  TARGET_LINK_LIBRARIES(${NAME} ${PROJECT_NAME} ${Boost_LIBRARIES})
+  TARGET_LINK_LIBRARIES(${NAME} PRIVATE ${PROJECT_NAME} ${Boost_LIBRARIES})
+  PKG_CONFIG_USE_DEPENDENCY(${NAME} roboptim-trajectory)
```

Comment:
If hpp-walkgen is correctly package, this last line shouldn't be necessary.

Reply:
I will try.

Reply:
You are right.